admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl

```diff
@@ -82,17 +82,19 @@
 		</div>
 	</div>
 
-	<div class="input-group">
-		<button type="button" class="btn btn-primary" tabindex="-1" id="import_continue_button" style="display: none;">
-			Ignore warnings and continue?
-		</button>
-		&nbsp;
-		<button type="button" class="btn btn-default" tabindex="-1" id="import_stop_button">
-			Abort import
-		</button>
-		&nbsp;
-		<button type="button" class="btn btn-success" data-dismiss="modal" tabindex="-1" id="import_close_button" style="display: none;">
-			Close
-		</button>
-	</div>
+  <div class="modal-footer">
+    <div class="input-group pull-right">
+      <button type="button" class="btn btn-primary" tabindex="-1" id="import_continue_button" style="display: none;">
+        Ignore warnings and continue?
```

Comment:
Yes, all the strings should be translatable, thanks @mickaelandrieu for spotting this.
translation domain: Admin.AdvParameters.Notification

Thank you!